package/gluon-status-page/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo

```diff
@@ -6,6 +6,6 @@ badrequest() {
         exit 1
 }
 
-( ubus call network.interface dump | jsonfilter -e "@.interface[@.proto='gluon_mesh' && @.up=true].device"| grep -qxF "$QUERY_STRING" ) 2>/dev/null || badrequest
+( /usr/bin/gluon-get-mesh-interfaces | grep -qxF "$QUERY_STRING" ) 2>/dev/null || badrequest
```

Comment:
The other absolute paths in these files don't make much sense either, let's not introduce another one.

Reply:
Sorry about changing my mind again, but I think I would prefer the verb "list" to "get" in the script name.

Reply:
done. - for the paths I had to touch the files again anyways. I like list better than get too.